gsadjust/data_analysis.py

```diff
@@ -112,49 +112,31 @@ def compute_gravity_change(obstreemodel, table_type='simple'):
                                                           role=256)  # 256=QtCore.Qt.UserRole
                     # Iterate through, look for matching station. 'if' statements deal with Gravnet, which truncates
                     # station names to 6 characters
-                    if len(initial_station.station) > 6 and len(station_name) > 6:
-                        if initial_station.station == station_name:
-                            break
-                    elif len(initial_station.station) > 6:
-                        if initial_station.station[:6] == station_name:
-                            break
-                    elif initial_station.station == station_name:
+                    if (initial_station.station == station_name or initial_station.station[:6] == station_name):
```

Comment:
@jkennedy-usgs This simplifies down these branches a bit. We have the following tests...

`if len(initial_station.station) > 6 and len(station_name) > 6  and initial_station.station == station_name`

However, `initial_station.station == station_name` is also tested last on the elsif, so whether the names are >6 or not, they will be compared for equality. This is simplified down to `initial_station.station == station_name` then we're left with the truncated test `initial_station.station[:6] == station_name`.

I don't *think* the order matters here, because if `initial_station.station[:6] == station_name` is true, then `station_name` must be <=6 characters.